.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+  "presets": [
+      "@babel/preset-react"
+  ],
+  "plugins": ["@babel/plugin-syntax-jsx"]
+}
```

Comment:
.gitignore file should be used to avoid committing unnecessary files. Please remove local linter files.